content/post/2020-02-07-processx-blocked-sigchld-fix/index.Rmd

```diff
@@ -0,0 +1,442 @@
+---
+slug: processx-blocked-sigchld
+title: "Debugging: Signals and Subprocesses"
+authors:
+  - Gábor Csárdi
+date: "2020-02-07"
+tags:
+- package development
+- debugging
+output: 
+  html_document:
+    keep_md: true
+---
+
+```{r include = FALSE}
+options(width = 60)
+```
+
+This is a short story about a non-trivial bug in the processx package,
+and how I fixed it. It is a good showcase of the some debugging tools.
```

Comment:
Since authors don't appear yet you should write your name after the first "I".

Reply:
Maybe we can wait until the names appear? :)